hbase-shell/src/main/ruby/shell/commands/flush.rb

```diff
@@ -25,10 +25,11 @@ def help
 Flush all regions in passed table or pass a region row to
 flush an individual region or a region server name whose format
 is 'host,port,startcode', to flush all its regions.
-You can also flush a single column family within a region.
+You can also flush a single column family within a table or region.
```

Comment:
`You can also flush a single column family for all regions within a table, or for an specific region only.`

Reply:
Will fix later.
Thanks.